web/themes/custom/cloudy/src/css/_bootstrap-overrides.scss

```diff
@@ -1,42 +1,23 @@
 /* Colors */
 
 // Assigns color names to specific hex values
-$blue: #418fde;
-$blue-dark: #1b5a99;
-$blue-darker: #0b4177;
-$blue-light: #e8eff5;
-$cyan: #17a2b8;
-$green: #046a38;
-$green-light: #e7f4e4;
-$red: #dc3545;
-$yellow: #ffb81c;
-$light-gray: #e8eff5;
-$gray: #e8eff5;
-$white: #fff;
-$black: #000;
+$blue: $cloudy-blue-base;
+$cyan: $cloudy-cyan-base;
+$green: $cloudy-green-base;
+$red: $cloudy-red-base;
+$yellow: $cloudy-yellow-base;
+$gray: $cloudy-gray-base;
+$white: $cloudy-white-base;
+$black: $cloudy-black-base;
 
 // Assigns primary, secondary, light, and dark to specific colors
-$primary: $blue-dark;
-$secondary: $blue-dark;
+$primary: $cloudy-blue-dark;
+$secondary: $cloudy-blue-dark;
 $light: $gray;
```

Comment:
$cloudy-gray, avoid chaining of definitions to enhance clarity of assigned value